.github/workflows/docs-push.yml

```diff
@@ -1,19 +1,58 @@
+name: Push docs to knowledgebase
 on:
   workflow_dispatch:
-  
+  pull_request:
+  push:
+    branches: main
```

Comment:
We are using the `master` branch instead of `main`

Reply:
fixed - think this is ready for re-review now